### ISSUE TITLE
step-63: Explain the relaxation factor

### ISSUE THREAD

**Issue by bangerth** [dealii/dealii#8223]
The introduction of step-63 explains the difference between additive and multiplicative Schwarz methods. It also briefly mentions SOR, but never introduces the relaxation factor. However, the relaxation factor then shows up in the input file. It might be worth to discuss where that factor appears in the formulas for the smoother.

@tjhei, @tcclevenger: FYI.

(As before, please wait for my update to step-63.)

**Comment by tjhei** [comment]
Fair point. @tcclevenger can you try to address this? I would be okay adding a paragraph at the end of the smoother discussion instead of inserting a relaxation parameter in all the formulas.

**Comment by tcclevenger** [comment]
Yes. I'll send off a pull request with all the changes discussed today.
